files/ja/learn/html/introduction_to_html/advanced_text_formatting/index.md

```diff
@@ -271,7 +271,7 @@ HTML には引用をマークアップするための機能もあります。ど
 ```html
 <p>
   The quote element — <code>&lt;q&gt;</code> — is
-  <q cite="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/q">
+  <q cite="https://developer.mozilla.org/ja/docs/Web/HTML/Element/q">
```

Comment:
This is a cite from the English page, so it should be the `/en-US/` link, I think.

Reply:
Most of the other locales also point to their own locale instead of to English, it looks like -- some also have translated these code blocks!